csharp-basic-topics/TaskCompletedTaskFromResultVsReturnInCSharp/TaskCompletedVsTaskFromResultVsReturnInCSharp/ReturnHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace TaskCompletedVsTaskFromResultVsReturnInCSharp;
+
+public class ReturnHandler
+{
+    public async Task<int> UseReturnAsync()
+    {
+        Console.WriteLine("About to perform some asynchronous work");
```

Comment:
In all your other `Console.Writeline` statements, you put the dot at the end of the sentence except here, let's add it here as well for consistency

Reply:
Ok. I have done this now. It will reflect in my next commit.